Sources/EdgeNetworkHandlers/NetworkResponseHandler.swift

```diff
@@ -146,8 +146,9 @@ class NetworkResponseHandler {
 
         for eventHandle in unwrappedEventHandles {
             let requestEventId = extractRequestEventId(forEventIndex: eventHandle.eventIndex, requestId: requestId)
-            if !ignoreStorePayloads {
+            if ignoreStorePayloads {
                 Log.debug(label: LOG_TAG, "Ignoring state:store payload for request with id: \(requestId)")
```

Comment:
Identities were reset recently, ignoring state:store payload for request with id: ...